ImagineAR/IAR-SDK-Sample-iOS
# IAR-5463 Update icons

### Overview

Update icons for Core, Surface and Target.

### Resources

IAR-5463

### Testing Instructions

Install Core, Surface and Target and see if the new icons (Text align to centre) and not the old icons (Text align to right)
